app/helpers/view_helper.rb

```diff
@@ -33,7 +33,11 @@ module ViewHelper
   end
 
   def viral_icon_source(name)
-    path = Rails.root.join('app', 'assets', 'icons', 'heroicons', "#{name}.svg")
+    path = if Rails.configuration.auth_config[name]
+             Rails.root.join('config', Rails.configuration.auth_config[name]) # Auth Icon Overrides
```

Comment:
**nitpick:** I think we should probably enforce a directory for these icons.  Maybe `config/auth/icon` or something

Reply:
Added.